### PR TITLE
feat: udsink grpc stream

### PR DESCRIPTION
**Pull request by chromevoid**
fixes: https://github.com/numaproj/numaflow/issues/335
Update tests based on https://github.com/numaproj/numaflow-go/pull/25

### CODE REVIEW

**Comment by chromevoid** [comment]
TODO: need to clean up all the numaio images and docs...

File: go.mod [hunk 1]
```diff
@@ -25,7 +25,7 @@ require (
 	github.com/montanaflynn/stats v0.0.0-20171201202039-1bf9dbcd8cbe
 	github.com/nats-io/nats-server/v2 v2.7.5-0.20220415000625-a6b62f61a703
 	github.com/nats-io/nats.go v1.21.0
-	github.com/numaproj/numaflow-go v0.2.4
+	github.com/numaproj/numaflow-go v0.2.5
```
Comment (vigith) [inline go.mod:28]: let's cut `0.3.0` for numaflow-go

File: test/sdks-e2e/README.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 Each SDK is supposed to implement:
 
-1. A `flatmap` UDF example, which splits the input message with `,`, and return a list. Build a docker image `quay.io/numaio/flatmap-example:${language}` and push to `quay.io`;
+1. A `flatmap` UDF example, which splits the input message with `,`, and return a list. Build a docker image `quay.io/numaio/numaflow-${language/map-flatmap:latest}` and push to `quay.io`;
```
Comment (vigith) [inline test/sdks-e2e/README.md:5]:
```suggestion
1. A `flatmap` UDF example, which splits the input message with `,`, and return a list. Build a docker image `quay.io/numaio/numaflow-${language}/map-flatmap:latest` and push to `quay.io`;
```

File: test/sdks-e2e/README.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
...
    
-2. A `simplesink` UDSink example, which prints out original message in pod logs. Build a docker image `quay.io/numaio/simplesink-example:${language}` and push to `quay.io`.
+2. A `simplesink` UDSink example, which prints out original message in pod logs. Build a docker image `quay.io/numaio/numaflow-${language/sink-log:latest` and push to `quay.io`.
```
Comment (vigith) [inline test/sdks-e2e/README.md:7]:
```suggestion
2. A `simplesink` UDSink example, which prints out original message in pod logs. Build a docker image `quay.io/numaio/numaflow-${language}/sink-log:latest` and push to `quay.io`.
```

File: go.mod (hunk 1 above)
Comment (chromevoid) [reply go.mod:28]: may I merge https://github.com/numaproj/numaflow-go/pull/26 and cut 0.3.0?

**Review by vigith — APPROVED** [review]

**Pull request merged**